modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
+[NOTE]
+====
+Having dots (`.`) in the database names is not recommended.
+This is due to the difficulty of determining if a dot is part of the database name or a delimiter for a database alias in a composite database.
+====
+
+.Query
+[source, cypher]
+----
+CREATE DATABASE customers
+----
+
+When a database has been created, it will show up in the listing provided by the command `SHOW DATABASES`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+----
++-------------+
+| name        |
++-------------+
+| "customers" |
+| "movies"    |
+| "neo4j"     |
+| "system"    |
++-------------+
+----
+
+[[manage-databases-existing]]
+=== Use `IF EXISTS` or `OR REPLACE` when creating databases
+
+Appending `IF NOT EXISTS` to the `CREATE DATABASE` command ensures that no error is returned and nothing happens should the database already exist.
+
+.Query
+[source, cypher]
+----
+CREATE DATABASE customers IF NOT EXISTS
+----
+
+Adding `OR REPLACE` to the command results in any existing database being deleted and a new one being created.
+
+.Query
+[source, cypher]
+----
+CREATE OR REPLACE DATABASE customers
+----
+
+This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.
+
+These commands are optionally idempotent, with the default behavior to fail with an error if the database already exists.
+
+These behavior flags apply to both standard and composite databases (e.g. a composite database may replace a standard one or another composite).
+
+[NOTE]
+====
+The `IF NOT EXISTS` and `OR REPLACE` parts of these commands cannot be used together.
+====
+
+[[manage-databases-create-database-options]]
+=== `CREATE DATABASE` options
+
+The `CREATE DATABASE` command can have a map of options, e.g. `OPTIONS {key: 'value'}`.
+
+[NOTE]
+====
+There are no available `OPTIONS` values for composite databases.
+====
+
+[options="header"]
+|===
+
+| Key | Value | Description
+
+| `existingData`
+| `use`
+|
+Controls how the system handles existing data on disk when creating the database.
+Currently this is only supported with `existingDataSeedInstance` and must be set to `use` which indicates the existing data files should be used for the new database.
+
+| `existingDataSeedInstance`
+| ID of the cluster server
+|
+Defines which server is used for seeding the data of the created database.
+The server ID can be found in the `serverId` column after running `SHOW SERVERS`.
+
+| `seedURI`
+| URI to a backup or a dump from an existing database.
+|
+Defines an identical seed from an external source which will be used to seed all servers.
+
+| `seedConfig`
+| comma separated list of configuration values.
+|
+Defines additional configuration specified by comma separated `name=value` pairs that might be required by certain seed providers.
+
+| `seedCredentials`
+| credentials
+|
+Defines credentials that needs to be passed into certain seed providers.
```

Comment:
```suggestion
Defines credentials that need to be passed into certain seed providers.
```

Reply:
I know some of my comments are on things that are copy pasted from cypher manual, but I'm noticing them now so XD